openpype/hosts/maya/plugins/publish/validate_vrayproxy.py

```diff
@@ -1,27 +1,33 @@
 import pyblish.api
 
+from openpype.pipeline import KnownPublishError
+
 
 class ValidateVrayProxy(pyblish.api.InstancePlugin):
 
     order = pyblish.api.ValidatorOrder
-    label = 'VRay Proxy Settings'
-    hosts = ['maya']
-    families = ['studio.vrayproxy']
+    label = "VRay Proxy Settings"
+    hosts = ["maya"]
+    families = ["vrayproxy"]
 
     def process(self, instance):
-
-        invalid = self.get_invalid(instance)
-        if invalid:
-            raise RuntimeError("'%s' has invalid settings for VRay Proxy "
-                               "export!" % instance.name)
-
-    @classmethod
-    def get_invalid(cls, instance):
         data = instance.data
+        self.log.info(data["family"])
+        self.log.info(data["families"])
```

Comment:
Not sure if it is neccessary to include the logging information....

Reply:
Good point! Its gone.